<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharsetSelector</class>
 <widget class="QWidget" name="CharsetSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="charsetCharsGroupBox">
     <property name="title">
      <string>Charset</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkCharsetUpper">
        <property name="toolTip">
         <string>ABCDEFGHIJKLMNOPQRSTUVWXYZ</string>
        </property>
        <property name="text">
         <string>Majuscules (A, B, C, ...)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="chkCharsetLower">
        <property name="toolTip">
         <string>abcdefghijklmnopqrstuvwxyz</string>
        </property>
        <property name="text">
         <string>Minuscules (a, b, c, ...)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkCharsetNumeric">
        <property name="toolTip">
         <string>0123456789</string>
        </property>
        <property name="text">
         <string>Chiffres (0, 1, 2, ...)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="chkCharsetSpace">
        <property name="toolTip">
         <string/>
        </property>
        <property name="text">
         <string>Espaces ( )</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="chkCharsetSpecial">
        <property name="toolTip">
         <string>!@#$%^&amp;*()-_+=</string>
        </property>
        <property name="text">
         <string>Spéciaux (!, $, #, ...)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Aussi inclure les caractères suivants : </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="txtCustomCharset">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
